src/eduid_webapp/reset_password/helpers.py

```diff
@@ -184,10 +184,11 @@ def send_password_reset_mail(email_address: str):
     to_addresses = [address.email for address in user.mail_addresses.verified.to_list()]
 
     pwreset_timeout = current_app.config.email_code_timeout // 60 // 60  # seconds to hours
+    # We must send the user to an url that does not correspond to a flask view,
+    # but to a js bundle (i.e. a flask view in a *different* app)
+    resetpw_link = f"{current_app.config.password_reset_link}code/{state.email_code.code}"
```

Comment:
Please use the urlappend function we have i eduid-common.api.utils to not have to worry about extra slashes.

Reply:
yep +1